src/main/java/net/vivialconnect/model/connector/ConnectorPaginatedPhoneNumbers.java

```diff
@@ -0,0 +1,61 @@
+package net.vivialconnect.model.connector;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class ConnectorPaginatedPhoneNumbers {
```

Comment:
Since the response always contains pagination, why not just add pagination to the existing class? This would make it cleaner and avoid breaking the code for some users.